src/deepsparse/evaluation/integrations/__init__.py

```diff
@@ -24,8 +24,7 @@ def try_import_lm_evaluation_harness(raise_error=False):
         if raise_error:
             raise ImportError(
                 "Unable to import lm_eval. "
-                "To install run 'pip install "
-                "git+https://github.com/EleutherAI/lm-evaluation-harness@b018a7d51'"
+                "To install run 'pip install lm-eval==0.4.0'"
```

Comment:
when or how will this error during normal use if `raise_error=False` by default? once the eval actually begins?

Reply:
I see. Good point.Yes, I will change the default behavior of this function, and set raise_error to True. 

This is the intended behavior when the acual eval is being ran. At runtime, when the user intends to use `lm-eval`, the module will try to do the hot import of the `lm-eval`. If it fails to find the dependency, installed, it will raise the error. 

However, when testing, I do not want to raise errors, but use the output of this function (boolean) to skip the tests that require lm-eval installed.